eventdiscovery-sdk/src/main/res/values/dimens.xml

```diff
@@ -13,7 +13,7 @@
     <dimen name="schedjoules_list_item_image_width">64dp</dimen>
     <dimen name="schedjoules_list_item_default_height">96dp</dimen>
     <dimen name="schedjoules_event_list_progress_bar">36dp</dimen>
-    <dimen name="schedjoules_event_detail_item_icon">28dp</dimen>
+    <dimen name="schedjoules_event_detail_item_icon">24dp</dimen>
```

Comment:
What's that? Height? Width? Margin? Padding? Spacing? Elevation? … Please add what kind of value this is.